Empire/Supply.cpp

```diff
@@ -163,6 +163,13 @@ bool SupplyManager::SystemHasFleetSupply(int system_id, int empire_id, bool incl
     return false;
 }
 
+bool SupplyManager::PlanetHasSupply(int planet_id) const {
+    std::shared_ptr<const Planet> planet = GetPlanet(planet_id);
+    if (!planet)
+        return false;
+    return (planet->Owner() == EmpireThatCanSupplyAt(planet->SystemID()));
```

Comment:
Although we do not seem likely to revert back to allowing multiple empires to supply a system, I hate to see that assumption be woven into code where it is not needed, especially since it looks like it would be a bit more efficient as
```    return SystemHasFleetSupply(planet->SystemID(), planet->Owner); ```

Or, since in the only place you are using this you already have the planet object available, perhaps instead of adding this new method at all you simply use ```SystemHasFleetSupply(planet->SystemID(), planet->Owner)``` in the SidePanel code